src/OSPSuite.Infrastructure/Configuration/OSPSuiteConfiguration.cs

```diff
@@ -65,21 +67,19 @@ protected OSPSuiteConfiguration()
          ApplicationSettingsFilePath = AllUsersFile(ApplicationSettingsFileName);
       }
 
-      private string retrieveProductDisplayName()
+      private bool isReleasedVersion(string version, int revision, string releaseDescription)
       {
-         var productDisplayName = $"{ProductNameWithTrademark} {MajorVersion}";
-         if (string.IsNullOrEmpty(ReleaseDescription))
-            return productDisplayName;
+         return string.Equals($"{version}.{revision}", releaseDescription);
+      }
 
-         return $"{productDisplayName} - {ReleaseDescription}";
+      private string retrieveProductDisplayName()
+      {
+         return _isReleasedVersion ? $"{ProductNameWithTrademark} {MajorVersion}" : $"{ProductNameWithTrademark} {ReleaseDescription}";
```

Comment:
We add the name in the header plus either X.Y or whateever is in Releas eDescription